res/css/style.css

```diff
@@ -602,20 +602,20 @@ body, #root, .profileViewer {
 .tabBarContainer {
   display: flex;
   flex-flow: row nowrap;
+  justify-content: space-between;
```

Comment:
so that the "sidebar" button is at the right while the tab bar is at the left. (maybe I should add this as a comment ?)

Reply:
OK I'll add it.